rust/flatbuffers/src/builder.rs

```diff
@@ -309,33 +310,6 @@ impl<'fbb> FlatBufferBuilder<'fbb> {
         WIPOffset::new(self.used_space() as UOffsetT)
     }
 
-    /// Create a vector by memcpy'ing. This is much faster than calling
```

Comment:
With the changes to `SafeSliceAccess` this method would have been relatively useless, I debated making a `ToByteSlice` trait to fill this role - as transmuting to byte arrays is perfectly safe, however, I opted to just fix `create_vector` so that it optimises to the same thing

Reply:
This is better - I don't think we should have features that force users to think about alignment or endianness of platforms